src/v2/ion/i18nTransformer.js

```diff
@@ -46,6 +46,8 @@ import Logger from 'util/Logger';
 import { getAuthenticatorDisplayName } from '../view-builder/utils/AuthenticatorUtil';
 import { FORMS, AUTHENTICATOR_KEY } from './RemediationConstants';
 
+const WEBAUTHN_API_GENERIC_ERROR_KEY = 'authfactor.webauthn.error';
```

Comment:
what is the confidence level that this won't change ?

Reply:
It would require a breaking change, and we have a comment to include the prefix for any new WebAuthn error strings, so the likelihood of it changing is probably quite low.